sensor_msgs/Image img_l
sensor_msgs/Image img_r
# sensor time
float32 ts_100us
# system epoch time
int64 nanoseconds
